### PR TITLE
placements annotations for multiple MIG devices

### PR DESCRIPTION
**Pull request by enoodle**

### CODE REVIEW

File: pkg/mig/config/config.go [hunk 1]
```diff
@@ -200,6 +201,7 @@ func (m *nvmlMigConfigManager) SetMigConfig(gpu int, config types.MigConfig) err
 			if mdt != valid {
 				return fmt.Errorf("unsupported MIG Device specified %v, expected %v instead", mdt, valid)
 			}
+
```
Comment (omer-dayan) [inline pkg/mig/config/config.go:204]: Lets remove? :)

File: pkg/mig/config/config.go [hunk 2]
```diff
@@ -200,6 +201,12 @@ func (m *nvmlMigConfigManager) SetMigConfig(gpu int, config types.MigConfig) err
...
+			info, _ := gi.GetInfo()
+			uuid, _ := info.Device.GetUUID()
+			cinfo, _ := ci.GetInfo()
+			fmt.Printf("EREZ DEBUG:: gi info: id: %d, ProfileId: %d, Placement: %v, uuid: %s\n", info.Id, info.ProfileId, info.Placement, uuid)
+			fmt.Printf("EREZ DEBUG:: ci info: id: %d, ProfileId: %d\n", cinfo.Id, cinfo.ProfileId)
```
[the remark this answers is not in the document]
Comment (enoodle) [reply pkg/mig/config/config.go]: :+1: was already removed

File: internal/nvml/mock.go [hunk 3]
```diff
@@ -348,6 +356,38 @@ func (d *MockA100Device) GetMigMode() (int, int, Return) {
...
+	if len(d.GpuInstances) == 0 {
+		return nil, MockReturn(ERROR_NOT_FOUND)
+	}
+	var gi *MockA100GpuInstance
+	for gi = range d.GpuInstances {
+		break
+	}
+	return gi, MockReturn(SUCCESS)
+}
+
```
[the remark this answers is not in the document]
Comment (enoodle) [reply internal/nvml/mock.go]: :+1: was already moved

File: cmd/export/placements.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package export
```
[the remark this answers is not in the document]
Comment (enoodle) [reply cmd/export/placements.go:1]: :+1: will have to move it to a different directory, and import it to the main file `cmd/export/export.go`

[the remark this answers is not in the document]
Comment (enoodle) [reply cmd/export/placements.go:1]: :+1:

File: pkg/mig/config/config.go (hunk 1 above)
Comment (enoodle) [reply pkg/mig/config/config.go]: :+1:

File: cmd/export/placements.go [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+	}
+	err = WriteOutput(os.Stdout, spec, f)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func ExportMigPlacements(c *Context) (map[int]map[int]string, error) {
```
[the remark this answers is not in the document]
Comment (enoodle) [reply cmd/export/placements.go]: :+1:

File: cmd/export/placements.go [hunk 6]
```diff
@@ -0,0 +1,35 @@
+package export
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/NVIDIA/mig-parted/cmd/util"
+)
+
+func ExportPlacements(c *Context, f *Flags) error {
```
[the remark this answers is not in the document]
Comment (enoodle) [reply cmd/export/placements.go]: :+1:

**Review by omer-dayan — APPROVED** [review]

**Pull request merged**